cf-revision-manager.php

```diff
@@ -2,200 +2,321 @@
 /*
 Plugin Name: CF Revision Manager
 Plugin URI: http://crowdfavorite.com
-Description: Revision management functionality so that plugins can add metadata to revisions as well as restore that metadata from revisions
-Version: 1.0.2
+Description: Revision management functionality so that plugins can add metadata to revisions as well as restore that metadata from revisions.
+Version: 1.0.1
 Author: Crowd Favorite
-Author URI: http://crowdfavorite.com 
+Author URI: https://crowdfavorite.com/
+License: GPL2
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+Domain Path: /languages
+Text Domain: cfrm
+ */
+
+// Exit if already accessed
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+* Fetch global variables declared for use anywhere in the plugin
+*
+* @since 1.0.1
+*
+* @return mixed Returns the global value depending on the key being provided
 */
-if (!class_exists('cf_revisions')) {
-	
-	define('CF_REVISIONS_DEBUG', false);
-	
-	function cfr_register_metadata($postmeta_key, $display_func = '') {
-		static $cfr;
-		if (empty($cfr)) {
-			$cfr = cf_revisions::get_instance();
-		} 
-		return $cfr->register($postmeta_key, $display_func);
+function cfrm_get_global( $key ) {
```

Comment:
I do not believe we need this `cfrm_get_global()` function. Currently the only place we use it is for enqueueing JS and CSS and we could simply use `plugin_dir_url( __FILE__ )` directly in all three instances.

If you have plans to leverage this in many other places throughout the plugin I'm willing to be persuaded to keep it. As is, though, this is unnecessary.